pkg/csi/service/cns/controller.go

```diff
@@ -53,8 +56,11 @@ type nodeManager interface {
 }
 
 type controller struct {
-	manager *common.Manager
-	nodeMgr nodeManager
+	manager            *common.Manager
+	nodeMgr            nodeManager
+	k8sInformerManager *k8s.InformerManager
```

Comment:
This makes the CSI driver k8s specific. Any easy way we can avoid this or at least limit it?

Reply:
As of now, this is the easiest way to get list of PVs and VolumeAtachments.